### DISCUSSION TITLE
Even after getting approved not yet received the GitHub student educational benefits.

### DISCUSSION THREAD

**Discussion by Ishanksinghdtu** [community/community#129702] [GitHub Education]
### Hi everyone,

When can I received the education pack for students.

**Comment by misapostle** [comment]
"We aim to provide you with your educational benefits within 3 days of this approval status. However, during busy periods like the start of a new semester, it may take a bit longer.

If you have not received your education benefits within 5 days of your approval status please raise a discussion on the [GitHub Education community](https://github.com/orgs/community/discussions/categories/github-education) and the Education community team will be able to assist in resolving this for you. 🌟"

it can even take weeks at busy times, just wait

**Reply by jig21nesh** [reply]
I am in the same boat. The application is approved, but he hasn't received the benefit. Also, I lost access to Copilot. I didn't expect that. (My current subscription was ending in two weeks, and I thought of renewing it before so that I wouldn't lose access.) 
Is there any way to reactive copilot access?
